app/subscriber/src/features/utils/castToSearchResult.ts

```diff
@@ -1,30 +1,10 @@
 import { ContentTypeName, IContentModel } from 'tno-core';
 
-export const castToSearchResult = (content: IContentModel) => {
+import { IContentSearchResult } from './interfaces';
+
+export const castToSearchResult = (content: IContentModel): IContentSearchResult => {
```

Comment:
@jdtoombs use interfaces properly so that we don't get regression issues.  Don't use anonymous interfaces.  If you do use them, only use them when they're not shared by anything other than one simple component.